ghost/core/core/server/models/mention.js

```diff
@@ -0,0 +1,18 @@
+const ghostBookshelf = require('./base');
+
+const Mention = ghostBookshelf.Model.extend({
+    tableName: 'mentions',
+
+    targetId: function targetId() {
```

Comment:
This should be renamed to `post` I think, but we can also leave the relation out of this PR so that we can get it merged and started on the repository implementation